op-program/scripts/build-prestates.sh

```diff
@@ -24,13 +25,65 @@ mkdir -p "${STATES_DIR}" "${LOGS_DIR}"
 
 cd "${REPO_DIR}"
 
-VERSIONS_JSON="[]"
-readarray -t VERSIONS < <(git tag --list 'op-program/v*' --sort taggerdate)
+function build_kona_prestate() {
+    local version=$1
+    if [[ -z "${version}" ]]; then
+        echo "Error: version is required"
+        exit 1
+    fi
+    local short_version
+    short_version=$(echo "${version}" | cut -c 14-)
+    local log_file="${LOGS_DIR}/build-kona-${short_version}.txt"
+    echo "Building Version: ${version} Logs: ${log_file}"
 
-for VERSION in "${VERSIONS[@]}"
-do
+    mkdir -p kona-prestate-build
+    cd kona-prestate-build
+
+    if [[ -d kona ]]; then
+        cd kona
+        git checkout --force "${version}" > "${log_file}" 2>&1
+    else
+        git clone -b "${version}" "$KONA_REPO_URL" kona > "${log_file}" 2>&1
+        cd kona
+    fi
+    mise trust --yes ./mise.toml
+
+    # HACK: The docker buildx version in cci is too old to parse variable blocks in docker-bake that contain
+    # descriptions. So patch them out
```

Comment:
If you build on a machine builder the description blocks don't cause issues. I think that would probably be better than patching these out.

We probably should consider updating the base image we use for docker builds in CI too. I'm not sure if there's any reason to stick with the old one (though maybe we don't do that right before a hard fork...)